cairo/ethereum/utils/bytes.cairo

```diff
@@ -195,3 +198,18 @@ func Bytes8_to_Bytes{range_check_ptr}(src: Bytes8) -> Bytes {
     tempvar res = Bytes(new BytesStruct(data=buffer, len=8));
     return res;
 }
+
+func Bytes_to_Bytes32{range_check_ptr}(src: Bytes) -> Bytes32 {
```

Comment:
Add assumption if no raise

Reply:
I'd like to clean a bytes ant uint lib once for all, maybe out of the scope of this PR

Reply:
also, we'll probably not reach prod with this codebase, but rather the M31 one, and there we'll have a range_check8 so these checks won't be useful anymore as bytes will be range checked